libraries/dialogs/src/main/java/com/trendyol/uicomponents/dialogs/BindingAdapters.kt

```diff
@@ -0,0 +1,21 @@
+package com.trendyol.uicomponents.dialogs
+
+import android.view.View
+import androidx.annotation.DrawableRes
+import androidx.appcompat.widget.AppCompatImageView
+import androidx.databinding.BindingAdapter
+
+@BindingAdapter("drawable")
+internal fun AppCompatImageView.setDrawable(@DrawableRes drawableResId: Int) {
```

Comment:
func name should be rename with setDrawableResource or any similar name

Reply:
Done.